config/routes.rb

```diff
@@ -1,21 +1,57 @@
 require 'sidekiq/web' if defined? Sidekiq
 require Rails.root.join('lib', 'router_constraints.rb')
 
-Ontohub::Application.routes.draw do
+Specroutes.define(Ontohub::Application.routes) do
 
   resources :filetypes, only: :create
 
   # IRI Routing #
   ###############
   #
+  # as per Loc/Id definition
+
+  ontology_subsites = %i(
+    mappings symbols children
+    sentences theorems comments
+    metadata ontology_versions graphs
+    projects categories tasks license_models formality_levels
+  )
+
+  ontology_subsites.each do |category|
+    specified_get "/:repository_id/*locid///#{category}" => "#{category}#index",
+      as: :"ontology_iri_#{category}",
+      constraints: [
```

Comment:
Style/AlignHash: Align the elements of a hash literal if they span more than one line.